src/NServiceBus.AcceptanceTests/Core/Recoverability/When_delayed_retries_with_regular_exception.cs

```diff
@@ -37,10 +39,9 @@ public RetryEndpoint()
             {
                 EndpointSetup<DefaultServer,Context>((config, context) =>
                 {
-                    config.EnableFeature<TimeoutManager>();
                     config.RegisterMessageMutator(new BodyMutator(context));
                     var recoverability = config.Recoverability();
-                    recoverability.Delayed(settings => settings.TimeIncrease(TimeSpan.FromMilliseconds(1)));
+                    recoverability.Delayed(settings => settings.NumberOfRetries(1).TimeIncrease(TimeSpan.FromMilliseconds(1)));
```

Comment:
Why `1`?

Reply:
Delayed retries are disabled by default, and the test needs them to be enabled to be valid. `1` seemed as good a value as any for that purpose.